src/main/java/org/sonar/plugins/github/MarkDownUtils.java

```diff
@@ -35,7 +35,7 @@
 @InstantiationStrategy(InstantiationStrategy.PER_BATCH)
 public class MarkDownUtils {
 
-  private static final String IMAGES_ROOT_URL = "https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/";
+  private static final String IMAGES_ROOT_URL = "http://sonarsource.github.io/sonar-github/";
```

Comment:
I was following GitHub "advice": this is the URL it shows in project settings. 
I am somewhat divided over this point. On one hand it may be more efficient for static images... On the other hand, I'm aware of the trend to put everything in https...